authn/management/commands/import_posts_to_dev.py

```diff
@@ -0,0 +1,136 @@
+import json
+import urllib.request
+from datetime import datetime, timedelta
+
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from posts.models.post import Post
+from users.models.user import User
+from common.markdown.markdown import markdown_text
+from utils.strings import random_string
+
+
+class Command(BaseCommand):
+    help = "Импорт постов с оригинального vas3k.club на dev/local сборки"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--pages",
+            type=int,
+            default=1,
+            help="Количество страниц, забираемых из фида",
+        )
+
+        parser.add_argument(
+            "--skip",
+            type=int,
+            default=0,
+            help="Количество страниц, которые надо пропустить",
+        )
+
+        parser.add_argument(
+            "--force",
+            action="store_true",
+            help="Заменять посты, если они уже существуют",
+        )
+
+    def handle(self, *args, **options):
+        if not settings.DEBUG:
+            return self.stdout.write("☢️  Только для запуска в DEBUG режиме")
+
+        result = {
+            'post_exists': 0,
+            'post_created': 0,
+            'user_created': 0
+        }
+
+        for x in range(options['skip'], options['pages'] + options['skip']):
+            url = "https://vas3k.club/feed.json?page={}".format(x + 1)
+            self.stdout.write("📁 {}".format(url))
+            req = urllib.request.Request(url)
+            req.add_header('User-Agent', 'Mozilla/5.0')
```

Comment:
Предлагаю поставить здесь отличимый User-Agent. На случай, если этот скрипт начнут абьюзить.

```suggestion
            req.add_header('User-Agent', 'poststodev')
```

Reply:
@igoose1 исправил на `posts-to-dev`